Sources/App/Core/Extensions/Array+queryString.swift

```diff
@@ -1,8 +1,10 @@
 
 extension Array where Element == QueryParameter {
-    func queryString(includeSeparator: Bool = true) -> String {
+    func queryString(includeSeparator: Bool = true, encoded: Bool = true) -> String {
         guard !isEmpty else { return "" }
-        let query: String = self.map { $0.encodedForQueryString }.joined(separator: "&")
+        let query: String = self
+            .map { encoded ? $0.encodedForQueryString : $0.unencodedQueryString }
```

Comment:
Nit: could perhaps call this just `$0.queryString` or push the `encoded` flag into `QueryParameter.queryString(encoded: Bool = false)`.